<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DragDropWidget</class>
 <widget class="QWidget" name="DragDropWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>視創用TGA/PNG轉檔工具</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="topButtonLayout">
    <item>
     <widget class="QPushButton" name="btn_tga_to_png">
      <property name="text">
       <string>TGA轉PNG</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_png_to_tga">
      <property name="text">
       <string>PNG轉TGA</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="topButtonSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="fileTable">
    <property name="columnCount">
     <number>3</number>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <column>
     <property name="text">
      <string>檔名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>大小</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>格式</string>
     </property>
    </column>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DropOnly</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="minimumHeight">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileActionLayout">
     <item>
      <widget class="QPushButton" name="btn_add_file">
       <property name="text">
        <string>添加檔案</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_remove_file">
       <property name="text">
        <string>移除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clear">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>闔於</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="outputGroup">
     <property name="title">
      <string>輸出設定</string>
     </property>
     <layout class="QVBoxLayout" name="outputLayout">
      <item>
       <widget class="QRadioButton" name="radio_current_folder">
        <property name="text">
         <string>保存在當前資料夾位置</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="customPathLayout">
        <item>
         <widget class="QRadioButton" name="radio_custom_folder">
          <property name="text">
           <string>自定位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_custom_path">
          <property name="text">
           <string>D:/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_browse">
          <property name="text">
           <string>[...]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_convert">
          <property name="text">
           <string>轉檔</string>
          </property>
          <property name="minimumWidth">
           <number>48</number>
          </property>
          <property name="maximumWidth">
           <number>48</number>
          </property>
          <property name="minimumHeight">
           <number>48</number>
          </property>
          <property name="maximumHeight">
           <number>48</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
    <!-- 移除底部轉檔按鈕 -->
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
